region/brinstar/blue/Construction Zone.json

```diff
@@ -90,11 +90,26 @@
       "name": "Geemer Farm",
       "requires": [
         "h_ZebesIsAwake",
-        {"resetRoom": {
-          "nodes": [1, 2]
-        }},
-        {"partialRefill": {"type": "Missile", "limit": 10}},
-        {"partialRefill": {"type": "Energy", "limit": 100}}
+        {"or": [
+          {"and": [
+            {"resetRoom": {"nodes": [1, 2]}},
```

Comment:
looks like `nodes` is the only thing that can exist inside of resetRoom. should this be collapsed to something like `{"resetRoom": [1, 2]}`? Or will there ever be anything else that could be needed in there (neighboring environment, runway length, etc)?